viewer/viewer-awt/pom.xml

```diff
@@ -1,73 +1,91 @@
-                </executions>
-            </plugin>
-        </plugins>
-    </build>
-
-</project>
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.icepdf.os</groupId>
+        <artifactId>viewer</artifactId>
+        <version>6.3.3-SNAPSHOT</version>
+    </parent>
+    <artifactId>icepdf-viewer</artifactId>
+    <packaging>jar</packaging>
+    <name>ICEpdf OS :: Viewer : Swing/AWT Viewer RI</name>
+    <description>
+        ICEpdf Java Swing/AWT reference implementation.
+    </description>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <issueManagement>
+        <system>github</system>
+        <url>https://github.com/pcorless/icepdf/issues</url>
+    </issueManagement>
+
+    <scm>
+        <connection>scm:git:https://github.com/pcorless/icepdf</connection>
+        <url>https://github.com/pcorless/icepdf/</url>
+    </scm>
+
+    <build>
+        <finalName>icepdf-viewer-${version}</finalName>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>templating-maven-plugin</artifactId>
+                <version>1.0.0</version>
+                <executions>
+                    <execution>
+                        <id>generate-verion-class</id>
+                        <goals>
+                            <goal>filter-sources</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>3.1.0</version>
+                <configuration>
+                    <source>8</source>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-source-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>attach-sources</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+    <dependencies>
+        <dependency>
+            <groupId>com.github.lookfirst</groupId>
+            <artifactId>sardine</artifactId>
+            <version>5.9</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>

```

Comment:
I think this is going to be deprecated pretty soon.  Can you use javax.xml.bind.* instead?

Reply:
Okay, I looked at [this](https://stackoverflow.com/questions/43574426/java-how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexceptio/43574427#43574427) post and added the dependencies.      
Not sure if there is a real difference between javax.xml.bind and jakarta.xml.bind.